circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -64,10 +62,8 @@ def execute_experiments(
         samplers: Sampler(s) on which to run the sub-experiments.
 
     Returns:
-        - A 3D list of length-2 tuples holding the quasi-distributions and QPD bit information
-          for each sub-experiment. The shape of the list is: (``num_unique_samples``, ``num_partitions``, ``num_commuting_observ_groups``)
-        - Coefficients corresponding to each unique subexperiment's
-          sampling frequency
+        - A list of :class:`~qiskit.primitives.SamplerResult` instances -- one for each partition.
```

Comment:
It's not a _list_; it's either a `dict` that maps each partition label to a `SamplerResult`, or else a single `SamplerResult`.